crudApp/src/actions/index.js

```diff
@@ -0,0 +1,32 @@
+//I have added four actions which is used for defining what actions we can take on performing operations on our Employees state.
+
+function addEmp(id, name, age) {
+  return {
```

Comment:
Please use the arrow functions as they are easy to read and  concise.
e.g
export const addEmp = (payload) => ({});

Replace all functions with this syntax